internal/venice-client-common/src/main/java/com/linkedin/venice/stats/routing/RoutingMetricEntity.java

```diff
@@ -0,0 +1,64 @@
+package com.linkedin.venice.stats.routing;
+
+import static com.linkedin.venice.stats.dimensions.VeniceMetricsDimensions.VENICE_HELIX_GROUP_ID;
+import static com.linkedin.venice.stats.dimensions.VeniceMetricsDimensions.VENICE_STORE_NAME;
+import static com.linkedin.venice.utils.CollectionUtils.setOf;
+
+import com.linkedin.venice.stats.dimensions.VeniceMetricsDimensions;
+import com.linkedin.venice.stats.metrics.MetricEntity;
+import com.linkedin.venice.stats.metrics.MetricType;
+import com.linkedin.venice.stats.metrics.MetricUnit;
+import com.linkedin.venice.stats.metrics.ModuleMetricEntityInterface;
+import java.util.Set;
+
+
+/**
+ * Metric entities for routing-related metrics.
+ */
+public enum RoutingMetricEntity implements ModuleMetricEntityInterface {
+  /**
+   * Count of available Helix groups for routing.
+   */
+  HELIX_GROUP_COUNT(
+      "helix_group.count", MetricType.MIN_MAX_COUNT_SUM_AGGREGATIONS, MetricUnit.NUMBER,
+      "Count of available Helix groups for routing", setOf(VENICE_STORE_NAME)
+  ),
+
+  /**
+   * Count of requests routed to each Helix group.
+   */
+  HELIX_GROUP_REQUEST_CALL_COUNT(
+      "helix_group.request.call_count", MetricType.COUNTER, MetricUnit.NUMBER,
```

Comment:
Can we just do call_count similar to call_time below?

Reply:
sure, fixed.